x/liquidstakeibc/types/keys.go

```diff
@@ -85,3 +94,7 @@ func GetValidatorUnbondingStoreKey(chainID, validatorAddress string, epochNumber
 func GetDepositStoreKey(chainID string, epochNumber int64) []byte {
 	return append([]byte(chainID), []byte(strconv.FormatInt(epochNumber, 10))...)
 }
+
+func GetLSMDepositStoreKey(chainID, delegatorAddress, denom string) []byte {
```

Comment:
why no epoch this time?

Reply:
no need for epoch, we track individual deposits
 
 at the end of the epoch we will just transfer all de PENDING deposits

Reply:
I will actually add it to differentiate between deposits that are already being processed and new ones (from the same validator and same token)